plugins/europeana/record.js

```diff
@@ -223,6 +223,7 @@ export default (axiosOverrides) => {
         media: this.aggregationMedia(providerAggregation, edm.type, edm.services),
         agents,
         concepts,
+        times,
```

Comment:
Here we return agents and concepts (as opposed to people and topics) so should also return timespans (as opposed to times).